tests/integration/test_cubepart.py

```diff
@@ -12,6 +12,18 @@
 class Describe_Slice(object):
     """Integration-test suite for _Slice object."""
 
+    def it_knows_cube_is_mr_by_itself(self):
+        cube = Cube(CR.EDU_FAV5_FAV5)
+        slice_ = cube.partitions[0]
+        assert slice_.table_name == cube.name
+        assert slice_.cube_is_mr_by_itself is True
+
+    def it_knows_cube_is_not_mr_by_itself(self):
+        cube = Cube(CR.AGE_FAVMR)
+        slice_ = cube.partitions[0]
+        assert slice_.table_name is None
+        assert slice_.cube_is_mr_by_itself is False
+
```

Comment:
These two are the same test with a different parameter. Parameterize a single test with `@pytest.mark.parametrize`, grep for examples in exporter if you need them.